src/sentry/seer/endpoints/seer_rpc.py

```diff
@@ -270,6 +270,9 @@ def get_organization_project_ids(*, org_id: int) -> dict:
 
 
 def _can_use_prevent_ai_features(org: Organization) -> bool:
```

Comment:
it seems not ideal that there are two copies of this function, can follow up on this